main/acle.rst

```diff
@@ -5822,3 +5835,195 @@ Instructions
 +---------------------------------+----------------+------------+-----------------+
 
 These intrinsics are available when ``arm_acle.h`` is included.
+
+Architectural Extension Bridges
+###############################
+
+.. _ssec-NEONSVEBridge:
+
+NEON-SVE Bridge
+===============
+
+
+The NEON_SVE Bridge adds intrinsics that allow conversions between NEON and
+SVE vectors.
+
+These intrinsics are defined in the header file ``arm_neon_sve_bridge.h``.
+
+``__ARM_NEON_SVE_BRIDGE`` is defined as 1 when ``arm_neon_sve_bridge.h`` is
+available.
+
+Including this header_file will include ``arm_neon.h`` and ``arm_sve.h``.
+
+``svset_neonq``
+---------------
+
+These intrinsics set the first 128 bits of SVE vector ``vec`` to ``subvec``.
```

Comment:
I've had a request to mention that passing `svundef` as the `vec` parameter allows the compiler to reuse the SVE register corresponding to the NEON register. Here is some suggested language, please can someone raise their hand if they can improve it :)

```suggestion
These intrinsics set the first 128 bits of SVE vector ``vec`` to ``subvec``.
When ``svundef`` is passed as the ``vec`` parameter, the compiler is able
reuse the SVE register overlapping the NEON input without generating
additional instructions.
```

Reply:
Sorry, I don' understand what is the issue that this change of wording is trying to address.

Reply:
My intent was to try and provide a hint to the user how they might use this. But maybe this is not an appropriate place to do that, so I am happy to drop this suggestion. I found it difficult to shoehorn it in here with appropriate language.

Reply:
The change looks good to me FWIW.  Might be worth making it a separate paragraph though.